Sources/BraveWallet/Extensions/BraveWalletExtensions.swift

```diff
@@ -227,4 +227,10 @@ extension BraveWallet.BlockchainToken {
   var assetBalanceId: String {
     "\(contractAddress)+\(symbol)"
   }
+  
+  var isAuroraSupportedToken: Bool {
+    let isSupportedContractAddress = WalletConstants.supportedAuroraBridgeTokensContractAddresses
+      .contains(where: { $0.caseInsensitiveCompare(contractAddress.lowercased()) == .orderedSame })
```

Comment:
Isn't `caseInsensitiveCompare` and `lowercased()` redundant? or is it just to be double safe